package.json

```diff
@@ -57,13 +58,12 @@
     "npm-keyword": "^4.1.0",
     "opn": "^3.0.2",
     "package-json": "^2.1.0",
-    "parse-help": "^0.1.1",
     "read-pkg-up": "^1.0.1",
     "repeating": "^2.0.0",
     "root-check": "^1.0.0",
     "sort-on": "^1.0.0",
     "string-length": "^1.0.0",
-    "tabtab": "^1.3.0",
+    "tabtab": "1.4.x",
```

Comment:
Let's keep using `^` range. The issue you had with 1.4 is that you published a breaking change without a major version bump which broke npm semver reliance.